multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1935,6 +1935,7 @@ def evaluate_coco(
         self._save_path = setup_save_path(
             old_save_path=self._save_path,
             model_loaded=self._model_loaded,
+            warn_if_exist=False,
```

Comment:
Shall we just enable this flag in the tutorial mode?

We can detect tutorial mode by detection `AUTOMM_TUTORIAL_MODE` in os.environ: https://github.com/awslabs/autogluon/blob/3f56287b66f8935f4f19450fd08788878bd40b73/multimodal/src/autogluon/multimodal/predictor.py#L387

Reply:
This should also be False if it's not in tutorial. We'll not create a new path or overwrite anything in `evaluate()` or `predict()`. And thus it's not necessary to give a warning